components/tooltip/index.css

```diff
@@ -203,23 +203,38 @@ governing permissions and limitations under the License.
     transform: translate(-50%, var(--spectrum-tooltip-tip-margin));
   }
 
-  &:hover .spectrum-Tooltip {
+  &:hover .spectrum-Tooltip,
+  &:focus .spectrum-Tooltip,
+  &.is-focused .spectrum-Tooltip,
+  *:focus .spectrum-Tooltip {
```

Comment:
This one here makes sense (show the tooltip if a child element of something that should show a tooltip has focus), but it will not active if a child element has `.is-focused` or `:hover`. was that intended?

Reply:
Again, this is for the interactive use case where the element receiving focus would be the descendant link.